provider/aws/docdb.go

```diff
@@ -209,7 +210,10 @@ func (r *DocumentDBManager) DescribeInstances() ([]*docdb.DBInstance, error) {
 		return nil, err
 	}
 
-	instances := []*docdb.DBInstance{}
+	if instances == nil {
+		instances = []*docdb.DBInstance{}
+	}
+
 	for _, instance := range resp.DBInstances {
 		instances = append(instances, instance)
```

Comment:
Please add the recursion function for Docdb.

Reply:
fixed